RaceControl/RaceControl/ChannelCollection.cs

```diff
@@ -0,0 +1,30 @@
+﻿using Prism.Mvvm;
+using RaceControl.Common.Enums;
+using RaceControl.Common.Interfaces;
+using System.Collections.ObjectModel;
+using System.Linq;
+
+namespace RaceControl
+{
+    public class ChannelCollection : BindableBase, IChannelCollection
+    {
+        public ChannelCollection(ObservableCollection<IPlayableChannel> allChannels, IPlayableChannel currentChannel)
+        {
+            CurrentChannel = currentChannel;
+
+            Onboards = new ObservableCollection<IPlayableChannel>(allChannels.Where(c => c.ChannelType == ChannelType.Onboard));
```

Comment:
Perhaps also sort drivers by driver number or last name? But we only get a single string here, `c.DisplayName`.